docs/int/faq/errors.mdx

```diff
@@ -130,7 +144,10 @@ docker-compose logs
         <h4>
           <code>Peer connection failing and dropping</code> error
         </h4>
-      </summary> Should be fixed in the latest version of Charon, upgrade and restart your node. Likely linked to Obol bootnode infrastructure. You can also try to [run your own bootnode](https://docs.obol.tech/docs/next/int/quickstart/group/quickstart-group-launchpad#self-host-a-bootnode).
+      </summary> 
+      Should be fixed in the latest version of Charon, upgrade and restart your node. 
```

Comment:
Can we take this text out/confirm whether or not this is fixed?

Reply:
done